x/bank/keeper/keeper.go

```diff
@@ -381,6 +397,11 @@ func (k BaseKeeper) UndelegateCoinsFromModuleToAccount(
 // MintCoins creates new coins from thin air and adds it to the module account.
 // It will panic if the module account does not exist or is unauthorized.
 func (k BaseKeeper) MintCoins(ctx sdk.Context, moduleName string, amounts sdk.Coins) error {
+	err := k.mintCoinsRestrictionFn(ctx, amounts)
+	if err != nil {
+		ctx.Logger().Error("Module %s attempted minting coins %s it did not have permission for", moduleName, amounts)
```

Comment:
```suggestion
		ctx.Logger().Error(fmt.Sprintf("Module %s attempted minting coins %s it did not have permission for", moduleName, amounts))
```

Reply:
Should we also add a `with error %v` and `err` in the log line?

Reply:
I think we definitely should